packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -131,6 +134,31 @@ export async function processOutboundDocumentQueryResps({
   }
 }
 
+function buildInterrupt({
+  requestId,
+  patientId,
+  cxId,
+  log,
+}: {
+  requestId: string;
+  patientId: string;
+  cxId: string;
+  log: typeof console.log;
+}) {
+  return async (reason: string): Promise<void> => {
+    log("[Programming error] " + reason + ", skipping DR");
+    capture.error("Programming error processing CQ Doc Retrieval", {
```

Comment:
nit: for consistency when searching through events/logs, should we use `[Programming error]` and `Doc retrieval` in both strings?

Reply:
One is for logs and the other is for Sentry, I was trying to keep readability based on each situation/UI. 

But I see your point and I'll make both share the same string.